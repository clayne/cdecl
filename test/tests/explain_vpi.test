cdecl | /dev/null | | explain int *volatile x | 0
