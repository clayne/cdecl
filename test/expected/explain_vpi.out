declare x as volatile pointer to int
